README.md

```diff
@@ -27,13 +27,17 @@ Report issues/feature requests on [GitHub Issues](https://github.com/ossfriday/o
 
 ### Environment variables
 
-```bash
-bundle exec figaro install
+Create a config file for the environment variables:
+
+```
+cp config/application.example.yml config/application.yml
 ```
 
-You'll need to set at least `github_client_id` and `github_client_secret` environment variables. The values can be obtained by [registering a new GitHub OAuth application](https://github.com/settings/applications/new) with the Authorization callback URL pointing to your application with `/users/auth/github/callback` appended.
+Register a [new GitHub OAuth application](https://github.com/settings/applications/new) with the Authorization callback URL pointing to `http://localhost:3000/users/auth/github/callback`.
 
-Or for more information about using figaro, see https://github.com/laserlemon/figaro
+Update `config/application.yml`, setting the `github_client_id` and `github_client_secret` variables generated for your GitHub OAuth Application
```

Comment:
Trailing `.` would be good. Also might want to be a little more forceful; this sounds like an instruction but maybe not a requirement. Would be good to have a `you will need to...` somewhere in the `### Environment variables` section.